bound/typescript/src/wasm/generated.d.ts

```diff
@@ -28,9 +28,136 @@ export function new_ed25519_signer(jwk: WasmJwk): WasmSigner;
 */
```

Comment:
Do I need to add this file to .gitignore?

Reply:
I'm indifferent on this at this moment, I can't think of a reason one way or the other